examples/tutorials/02_pipeline_operations.ipynb

```diff
@@ -15,7 +15,6 @@
    "source": [
```

Comment:
<p>Too complicated for such a simple idea.</p><p> </p><p>Suggestion:</p><p> </p><p><code>pipeline = dataset.p.func("main")</code></p><p><code>pipeline.before.func("before")</code></p><p><code>pipeline.after.func("after")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>